spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe Admin::VideosController do
+  it {is_expected.to use_before_action(:require_user)}
+  
+  describe "GET new" do
+    before {set_current_user}
```

Comment:
Thanks. Yeah, that was pointless for me to use the before. Completely understand. Thanks for calling me on it! :)